app/_kong_plugins/acl/examples/allow-consumer-groups.yaml

```
@@ -0,0 +1,18 @@
+description: 'Allows the `dev` and `admin` Consumer Groups on the Service or Route that the ACL plugin is enabled on.'
```

This should be both Consumers and Consumer Groups:

```suggestion
description: 'Allows the `dev` and `admin` Consumers and Consumer Groups on the Service or Route that the ACL plugin is enabled on.'
```